v1/slogger/logger_test.go

```diff
@@ -156,22 +156,10 @@ func TestFilter(test *testing.T) {
 }
 
 func TestStacktrace(test *testing.T) {
-	// slogger/logger_test.go:129
-	// testing/testing.go:346
-	// runtime/proc.c:1214
-
 	stacktrace := NewStackError("").Stacktrace
 	if match, _ := regexp.MatchString("^at v1/slogger/logger_test.go:\\d+", stacktrace[0]); match == false {
 		test.Errorf("Stacktrace level 0 did not match. Received: %v", stacktrace[0])
 	}
-
```

Comment:
This part of the test was removed because it turned out to be dependent on the version of Go you use.